apps/server/src/entities/user.entity.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+
+import { User } from './user.entity';
+
+describe('User Entity', () => {
+	describe('constructor', () => {
+		it('should throw an error by empty constructor', () => {
+			// @ts-expect-error: Test case
+			const test = () => new User();
+			expect(test).toThrow();

```

Comment:
why not checking the error type and that it is thrown when a single required param is missing and we get a concrete error with details about it?
Can this happen?!